nebula-logger/main/logger/aura/logJSONViewer/logJSONViewerHelper.js

```diff
@@ -36,4 +36,4 @@
         document.execCommand('copy');
         document.body.removeChild(hiddenJSONInput);
     }
-})
\ No newline at end of file
+});
```

Comment:
Weird.... I don't think I've ever put a `;` at the end like this, but I guess that is the right way to do it here. Works for me.

Reply:
you can disable things like that in `.prettierrc`, but in general outside of the `printWidth` (because new line after 80 characters is a _little_ too strict) I tend to simply follow the defaults recommended by SF in the lwc-recipes trailhead repo.